lib/platform/bitbucket-server/index.md

```diff
@@ -53,3 +52,9 @@ You should then receive a "Configure Renovate" onboarding PR in any projects tha
 
 We support all Bitbucket Server versions which are not EOL.
 See [Atlassian Support End of Life Policy](https://confluence.atlassian.com/support/atlassian-support-end-of-life-policy-201851003.html#AtlassianSupportEndofLifePolicy-BitbucketServer) for uptodate versions.
+
+## Assignees support
+Assignee support is added in version 17.2.0 of Renovate.
```

Comment:
```suggestion

Assignee support was added in version `17.2.0` of Renovate.
```

Use monospaced style for versions.
Add a newline after a heading.
I think we can use the past tense here, as the version was release a while back. 😉